src/main/java/com/google/api/codegen/viewmodel/ApiMethodView.java

```diff
@@ -14,4 +14,6 @@
  */
 package com.google.api.codegen.viewmodel;
 
-public interface ApiMethodView {}
+public interface ApiMethodView {
+  String name();
```

Comment:
we use the `getX()` naming pattern in general - view models are the exception.

Reply:
Is this just an FYI? This is a view model so this method name is fine, then, right?

Reply:
Oh sorry, I didn't notice.